src/modules/recommendations.js

```diff
@@ -1,4 +1,4 @@
-/* eslint-disable object-curly-newline, no-param-reassign */
+/* eslint-disable object-curly-newline, no-param-reassign, no-underscore-dangle */
```

Comment:
Do we still need this `no-underscore-dangle`? I'm guessing you removed some code that needed this

Reply:
Thanks for catching this! I think I added it for something else and then removed it. Will be removing the rule override 👍